src/Categories/Category/Dagger/Instance/Rels.agda

```diff
@@ -1,24 +1,37 @@
-{-# OPTIONS --without-K --safe #-}
-module Categories.Category.Dagger.Instance.Rels where
+{-# OPTIONS --safe --without-K #-}
 
-open import Data.Product
-open import Function
-open import Relation.Binary.PropositionalEquality
-open import Level
+module Categories.Category.Dagger.Instance.Rels where
 
 open import Categories.Category.Dagger
 open import Categories.Category.Instance.Rels
 
+open import Data.Product
```

Comment:
Please put explicit `using` in as many places as possible, strongly so on stdlib imports.